runtime/minor_gc.c

```diff
@@ -485,8 +487,8 @@ void caml_alloc_small_dispatch (tag_t tag, intnat wosize, int flags)
   while(1) {
     /* We might be here because of an async callback / urgent GC
        request. Take the opportunity to do what has been requested. */
-    if (flags & CAML_FROM_CAML) caml_check_urgent_gc (Val_unit);
```

Comment:
Why removing this call? If an urgent minor or major collection has been requested but the minor heap is not empty, this is the only entry point for the GC.

Reply:
It is still there right below.

Reply:
Ah, sorry for that, I misread.